crates/web-plugins/didcomm-messaging/src/midlw.rs

```diff
@@ -133,7 +133,9 @@ async fn unpack_payload(
         return Err(response.into_response());
     }
 
-    if plain_message.from.is_none() || !metadata.authenticated || metadata.anonymous_sender {
+    if plain_message.type_ != "https://didcomm.org/routing/2.0/forward"
```

Comment:
I suggest that you define that string literal as a `const` for better readability.
```suggestion
    if plain_message.type_ != ROUTING_2_0_MSG_TYPE
```